ui/src/app/pages/roles/components/modals/grantAccessModal.tsx

```diff
@@ -320,7 +347,7 @@ export class GrantAccessModal extends PureComponent<GrantAccessModalProps, Grant
 
```

Comment:
While working on updates for this, one of our content designers provided a new description to make the text more consistent with other text we're using for the button label and modal header.

Can you replace the following text:
> Manage access to resources in this Service Registry instance by assigning permissions to an account.
With this text:
> Grant access to resources in this Service Registry instance by assigning a role to an account.

Can you also replace this text:
> Manage access to resources in the Registry by assigning permissions to an account.
With this text:
> Grant access to resources in the Registry by assigning permissions to an account

Reply:
done